src/main/java/hudson/plugins/sloccount/SloccountResult.java

```diff
@@ -25,7 +25,7 @@
 
     private transient SloccountReport report;
 
-    private final AbstractBuild<?,?> owner;
+    private transient final Run<?,?> owner;
```

Comment:
I have checked - it must be transient as Run<> is not serializable.
However field is needed due as it is referenced from jelly file (side panel).
I will add setter and properly initialize it from onLoad in BuildAction